core/kernel/tee_ta_manager.c

```diff
@@ -626,7 +627,7 @@ static void update_current_ctx(struct thread_specific_data *tsd)
 	if (((ctx && is_user_ta_ctx(ctx) ?
 			to_user_ta_ctx(ctx)->mmu : NULL) == NULL) ==
 					core_mmu_user_mapping_is_active())
-		panic();
+		panic("unexpected active mapping");
```

Comment:
You should replace `#define panic(...) __do_panic(__FILE__, __LINE__, __func__, (void *)0)` with `#define panic() __do_panic(__FILE__, __LINE__, __func__, (void *)0)` so that those mistakes are detected at compile time.